esignet-default.properties

```diff
@@ -199,14 +199,14 @@ mosip.esignet.ida.vci-user-info-cache=userinfo
 
 mosip.esignet.cache.names=clientdetails,preauth,authenticated,authcodegenerated,userinfo,linkcodegenerated,linked,linkedcode,linkedauth,consented,authtokens,bindingtransaction,vcissuance
 
-#spring.cache.type=redis
-#spring.cache.cache-names=${mosip.esignet.cache.names}
-#spring.redis.host=localhost
-#spring.redis.port=6379
+spring.cache.type=redis
+spring.cache.cache-names=${mosip.esignet.cache.names}
+spring.redis.host=localhost
+spring.redis.port=6379
 management.health.redis.enabled=false
-
+spring.redis.password=mypass
```

Comment:
password should not be hardcoded, please set it as a placeholder to config-server